hw/ip/prim/util/primgen.py

```diff
@@ -45,6 +44,7 @@ def _filter_primitives(core):
         if (vlnv['vendor'] == 'lowrisc' and
                 vlnv['library'].startswith('prim_') and
             (prim_name is None or vlnv['name'] == prim_name)):
+
```

Comment:
Huh, that's interesting: I didn't know a newline would placate flake8 for this. Thanks.

Reply:
It doesn't, unfortunately; the current code isn't flake8-clean. There is a bit of a disagreement between me, flake8, and yapf going on that I was unable to resolve to everyone's satisfaction :smile:  So I went for a solution that retains the spirit of the flake8 warning (same indentation levels are confusing), but only helps the human, not flake8.

(In this case, the second line of the condition is forced to be like that by yapf even though I'd indent it less, and the four space indent of the third line of the condition is forced by yapf as well, which is what I'd use as well.)

Reply:
Oof. Well, I think I agree with your notes about style. In the past, I've done this to placate flake8:

```python
if ((foo and
     bar)):
    something_indented_by_4
```
In this example, it looks a bit silly but with real code it actually looks alright. Maybe my lisp background has just inured me to parenthesis overload :-)

Reply:
> Oof. Well, I think I agree with your notes about style. In the past, I've done this to placate flake8:
> 
> ```python
> if ((foo and
>      bar)):
>     something_indented_by_4
> ```
> 
> In this example, it looks a bit silly but with real code it actually looks alright. Maybe my lisp background has just inured me to parenthesis overload :-)

Just a note that my recent PR (draft, #2743) has lots of conflicting requests from human, yapf, and flake8. I've done none of them so as to only highlight the intent of the PR, but I (for one, but obviously more removed from the trenches) am not sure which to listen to. I can file an issue but I imagine there is one already. The closest I could find was #1909.